cli/src/base.ts

```diff
@@ -162,6 +162,16 @@ export abstract class BaseCommand extends Command {
       if (result.success) {
         this.projectConfig = result.data;
         this.projectConfigLocation = search.filepath;
+
+        // Temporal, to be removed in the future when everyone has migrated to the new format
+        if (this.projectConfig.databaseURL) {
+          const { region } = this.parseDatabaseURL(this.projectConfig.databaseURL);
+          if (region === 'eu-west-1' && !this.projectConfig.databaseURL.includes('eu-west-1')) {
```

Comment:
I guess we are using `eu-west-1` as a default in `parseDatabaseURL`. What if we remove the default and just check if `region` is not defined?

Reply:
That would affect also to the SDK, are we good with that?

Reply:
Yep. I think it's a good idea to make it mandatory from now on.